views/content/token.md

```diff
@@ -15,12 +15,13 @@ This is the code for the contract we're building:
         mapping (address => uint) public coinBalanceOf;
         event CoinTransfer(address sender, address receiver, uint amount);
       
-      /* Initializes contract with initial supply tokens to the creator of the contract */
-      function token(uint supply) {
-            coinBalanceOf[msg.sender] = (supply || 10000);
+        /* Initializes contract with initial supply tokens to the creator of the contract */
+        function token(uint supply) {
+            if (supply == 0) supply = 10000;
```

Comment:
I had tested it and the previous way of doing it worked and is shorter.. Why do you prefer it like this?